tutorials/cp-starter-integration-cpi-onboard-subscribe/cp-starter-integration-cpi-onboard-subscribe.md

```diff
@@ -1,79 +1,98 @@
+To provision your SAP Cloud Platform Integration tenant, you must first subscribe to the Process Integration service, assign the necessary roles to your user, and then provision your SAP Cloud Platform Integration tenant.
 
 ---
 
-[ACCORDION-BEGIN [Step 1: ](Subscribe to Process Integration service)]
-1. Login to the [SAP Cloud Platform Cockpit](https://cockpit.hanatrial.ondemand.com/cockpit/#/home/trialhome) and choose **Enter your Trial Account**. If this is your first time accessing the trial account, the cockpit will create a new trial account for you. Please wait for the process to complete and then click on **Enter your Trial Account** again.
+[ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-    ![Access trial account](1.0.0.create-cp-subaccount.png)
+1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
 
-    You see a subaccount with name **trial** already created. Select this subaccount.
+2. Navigate to the trial global account by clicking **Enter Your Trial Account**.
 
-    ![Access subaccount](1.0.1.access-subaccount.png)
+    !![Trial global account](01_Foundation20Onboarding_Home.png)
 
-2. In your subaccount, select **Subscriptions** > **Process Integration**.
+    >If this is your first time accessing your trial account, you'll have to configure your account by choosing a region (select the region closest to you). Your user profile will be set up for you automatically.  
+
+    >Wait till your account is set up and ready to go. Your global account, your subaccount, your organization, and your space are launched. This may take a couple of minutes.
+
+    >Choose **Continue**.
+
+    >![Account setup](02_Foundation20Onboarding_Processing.png)
+
+3. From your global account page, choose the `trial` tile to access your subaccount.
+
+
+4. Select **Subscriptions** > **Process Integration**.
 
     ![Select process integration service](1.1.select-process-integration.png)
 
-3. Select the **Subscribe** button.
+5. Select **Subscribe**.
 
     ![Select subscribe button](1.2.select-subscribe.png)
 
-    Wait for the process to complete. You will see the status change to **Subscribed** in green.
+    Wait for the process to complete. The status changes to **Subscribed** in green.
 
-    ![Subscription confirmation for Process Integratiin](1.2.1.pi-subscribed-confirmation.png)
+    ![Subscription confirmation for Process Integration](1.2.1.pi-subscribed-confirmation.png)
 
 [DONE]
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 2: ](Assign required roles)]
+>**IMPORTANT**:
+ With this step, you define permissions for users accessing SAP Cloud Platform Integration. For different tasks, different role templates are predefined, each role template containing the permissions for a set of tasks done by a typical persona such like the integration developer. We propose that you define an own role template that contains the most important role templates predefined by SAP.
```

Comment:
This sentence is really long, which makes it hard to read. Could you please split it into two sentences?

```suggestion
 With this step, you define permissions for users accessing SAP Cloud Platform Integration. For different tasks, different role templates are predefined. Each role template contains the permissions for a set of tasks done by a typical persona such like the integration developer. We propose that you define an own role template that contains the most important role templates predefined by SAP.
```

Reply:
Corrections added